permissions/component-async-http-client.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "async-http-client"
+github: "jenkinsci/async-http-client"
+paths:
+- "com/ning/async-http-client" # For the 1.7.x branch which is the only needed one at the moment
```

Comment:
Didn't we historically change groupId for forked libraries?

Reply:
I don't have such context. More than happy to do it if that is the trend.

Reply:
@oleg-nenashev might know.

I think either this, or going with a `1.2-jenkins-3` style version number.

Reply:
No, usually we do not change unless there is shading/class relocation involved.
Regarding versioning, `1.2-jenkins-3` is what I would expect as well